utils/membership_notifications.py

```diff
@@ -0,0 +1,32 @@
+from notifications.actions import push_notification
+
+from groups.utils.get_category import get_category
+
+
+def send_membership_notification(application_name, membership_type, person_id):
+    """
+    send appropriate notifications to the application member as per
+    the membership_type parameter
+    :param application_name: name of the application person is added/removed
+    :param membership_type: add/remove
+    :param person_id: id of the person added/removed
+    :return: notification
+    """
+
+    if membership_type == 'add':
+        template = f'You are added in {application_name}'
```

Comment:
Change to past tense and make it a little more detailed like:
```suggestion
        template = f'You were added to the developer group for the application \'{application_name}\''.
```